src/main/java/org/symphonyoss/symphony/messageml/bi/BiItem.java

```diff
@@ -68,4 +68,34 @@ protected void increaseAttributeCount(String attributeName) {
               attributeName, getName());
     }
   }
+
+  /**
+   * If the attribute is found inside the map this method will set the corresponding value with the max found between the
+   * current one and the one in input, otherwise the attribute will be put in the map with the attribute value passed as input.
+   * If the value found is not an integer, then no update will be performed
+   *
+   * @param attributeName name of the attribute to be checked
+   * @param attributeValue value of the attribute to be checked
+   */
+  protected void setMaxAttribute(String attributeName, Object attributeValue) {
```

Comment:
should we make attributeValue parameter an integer then?

Reply:
Done in https://github.com/symphonyoss/messageml-utils/pull/266/commits/bedab46386b8a9405936838f80189140567d89d4